### DISCUSSION TITLE
Lossless Animated WEBP

### DISCUSSION THREAD

**Discussion by Jay123210599** [ImageMagick/ImageMagick#7705] [Help]
### ImageMagick version

7.1.1-27

### Operating system, version and so on

Windows 11

### Description

Can ImageMagick make lossless animated webp files? If so, how?

**Comment by snibgo** [comment]
As with your other question https://github.com/ImageMagick/ImageMagick/discussions/7700:
```
magick frame*.png -quality 100 -define webp:lossless=true out.webp
```
If the inputs are 8 bits per channel, the output seems to be identical.

**Reply by Jay123210599** [reply]
What do you mean _seems_ to be?

**Comment by snibgo** [comment]
It means that I don't know if it is always lossless.
